<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POUInfoDialog</class>
 <widget class="QDialog" name="POUInfoDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>328</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>POU Info</string>
  </property>
  <widget class="QWidget" name="horizontalLayout">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>429</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="POUI_QHaboxLayout_okcancel">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="POUI_okButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="POUI_cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="POUI_tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>308</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="POUI_InfoTab">
    <attribute name="title">
     <string>POU Info</string>
    </attribute>
    <widget class="QTextEdit" name="POUI_textEdit_comment">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>148</y>
       <width>291</width>
       <height>195</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>291</width>
       <height>195</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>291</width>
       <height>195</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="documentTitle">
      <string/>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="tabStopWidth">
      <number>50</number>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>299</width>
       <height>132</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="POUI_GridLayout_inputfields">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="POUI_VBoxLayout_project">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="POUI_label_Prj">
          <property name="text">
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="POUI_lineEdit_prj"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="POUI_VBoxLayout_version">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="POUI_label_Version">
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="POUI_lineEdit_version"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="POUI_VBoxLayout_name">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="POUI_label_Name">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="POUI_lineEdit_name"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="POUI_VBoxLayout_plc">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="POUI_label_PLC">
          <property name="text">
           <string>PLC Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="POUI_lineEdit_plc"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="POUI_VBoxLayout_developer">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="POUI_label_Developer">
          <property name="text">
           <string>Developer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="POUI_lineEdit_developer"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="POUI_VBoxLayout_date">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="POUI_label_Date">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="POUI_lineEdit_date"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="POUI_checkBox_Comment">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>350</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>copy edit field into instruction section</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="POUI_PrototypeTab">
    <attribute name="title">
     <string>POU Prototype</string>
    </attribute>
    <widget class="QTextEdit" name="POUI_textEdit_Prototype">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>2</y>
       <width>292</width>
       <height>365</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>292</width>
       <height>365</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>292</width>
       <height>365</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>POUI_cancelButton</tabstop>
  <tabstop>POUI_okButton</tabstop>
  <tabstop>POUI_tabWidget</tabstop>
  <tabstop>POUI_lineEdit_prj</tabstop>
  <tabstop>POUI_lineEdit_version</tabstop>
  <tabstop>POUI_lineEdit_name</tabstop>
  <tabstop>POUI_lineEdit_plc</tabstop>
  <tabstop>POUI_lineEdit_developer</tabstop>
  <tabstop>POUI_lineEdit_date</tabstop>
  <tabstop>POUI_textEdit_comment</tabstop>
  <tabstop>POUI_checkBox_Comment</tabstop>
  <tabstop>POUI_textEdit_Prototype</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>POUI_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>POUInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>POUI_okButton</sender>
   <signal>clicked()</signal>
   <receiver>POUInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
